samples/GraphQL.DataLoader.Sample.Default/GraphQl/SalespersonGraphType.cs

```diff
@@ -0,0 +1,25 @@
+using GraphQL.DataLoader.Di.Sample.Types;
+using GraphQL.Types;
+using Microsoft.EntityFrameworkCore;
+
+namespace GraphQL.DataLoader.Di.Sample.GraphQl;
```

Comment:
namespace needs to match project

Reply:
easiest way to fix it here is to right-click the "Di" and rename it, and it will change all the files